Parsing fixes

tomer953 opened the pull request:
- it seems like the 'strong' class inside getTotalPageElement() is not longer needed due to UI changes
- fix prices with comma seperator (ie: amazon it) being parsed as one int (ie 123,45 -> 12345)

tomer953 committed:
fix comma prices bug, fix getTotalPageElement selector

tomer953 committed:
version bump

yfrans commented on script.js:
```diff
@@ -308,7 +308,7 @@ function extractPrice(element) {
         newPrice = price.replace('.', '').replace(',', '.');
     } else {
-        newPrice = price.replace(',', '');
+        newPrice = price.replace(',', '.');
```
@tomer953 Did you check it on multiple currencies?
If not, please check amazon.jp, co.uk, de, fr and others just to make sure this change is not breaking the behavior.

tomer953 replied:
I checked com, uk, it, de

i will update about jp tomorrow

tomer953 replied:
JP appears as "0", I don't think it related to this commit, but I'll try to fix it as well later today

![image](https://user-images.githubusercontent.com/1807493/107003103-eb928180-6794-11eb-8e0b-202e3b638eb9.png)

tomer953 replied:
OK found it, it is related to this commit,
since in JP and IT the comma is used differently, probably you already knew this :)
so hold on, I'll figure it out...

yfrans replied:
"probably you already knew this :)"

yep 😆

tomer953 replied:
I think we must base on the 'host' (ie: co.uk \ .de \ .fr etc) since the currency itself is not enough to figure out the thousand seperator used in that country, for example, italy spain and france are all using Euro but each one of them using different thousand seperator (comma, dot, space)

so I wrote a little function to handle it, Ill give it a try and will update soon

tomer953 committed:
parse currency by country

tomer953 commented:
I added new commit to parse currency by country...

I checked all the domains in the manifest, looks fine...

tomer953 commented:
after a little more testing I found out that some sites, like .de
use 'dot' seperator on the product view, and search view, but in the checkout view the seperator changes to 'comma'
so even inside a specifc host we have different rules between pages.. damn.. ill try to solve it


**Edit:**
I did, added another commit which solved the checkout page problem on EURO and .de
also fixed the selector on the search page to make it work again as well

Check all the domains and seems fine, I think we are ready to merge, in order to let the comminuty find more bugs

tomer953 committed:
fix search page, fix checkout page for .de

yfrans commented on script.js:
```diff
@@ -330,4 +330,35 @@ function formatPrice(price) {
+    // for: | .it | .es | .fr
+    if (site.endsWith('.it') || site.endsWith('.es') || site.endsWith('.fr')) {
+        // avoid for ils in checkout page
+        if (currency != 'ILS') {
```
Why we need a special condition for ILS? don't we have this issue in other currencies?
Merging anyway, but we need to check it.
Thanks :)

tomer953 replied:
this condition solves the following problem:

![image](https://user-images.githubusercontent.com/1807493/107042033-f0255d00-67c9-11eb-8604-5b0ce365c4fe.png)


the host is amazon.it,  but the currency is ILS
I'm sure theres a smarter way to map all behaviour, but lets start with something... 
I'm still learning all the different possibilites

also in the future I think we need to disable converation to the same rate (ILS -> ILS) as shown in this image
